docs/layer_reference/utility.rst

```diff
@@ -32,6 +32,13 @@ Print Layer
     :members:
     :undoc-members:
 
+Remove Layer
+------------
+
+.. autoclass:: returnn.tf.layers.basic.RemoveLayer
+    :members:
+    :undoc-members:
```

Comment:
This is also a layer which probably should be deprecated. Have you ever used it? The layer class name is also ambiguous, i.e. it is not immediately clear what it does. Maybe we should rename it. Or remove it, as you can do the same with `MaskingLayer`, in a more generic and clear way.